lib/elixir/lib/access.ex

```diff
@@ -236,6 +236,23 @@ defmodule Access do
     :error
   end
 
+  @doc """
+  Same as `fetch/2` but returns the term directly,
+  or raises an `ArgumentError` exception if an error is returned.
+  """
+  @spec fetch!(container, term) :: term
+  def fetch!(container, key) do
+    case fetch(container, key) do
+      {:ok, value} ->
+        value
+
+      :error ->
+        raise ArgumentError,
```

Comment:
Please see how `fetch!` is implemented in Keyword. The best is to return `raise KeyError`.

Reply:
Done!